src/advanced_fees_management.tf

```diff
@@ -0,0 +1,178 @@
+# region
+resource "azurerm_resource_group" "advanced_fees_management_rg" {
+  name     = format("%s-afm-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+# local variables
+locals {
+  advanced_fees_management_cors_configuration = {
+    origins = ["*"]
+    methods = ["*"]
+  },
+  advanced_fees_management_cosmosdb_enable_serverless = contains(var.advanced_fees_management_cosmosdb_extra_capabilities, "EnableServerless")
+}
+
+# subnet
+module "advanced_fees_management_snet" {
+  count                                          = var.advanced_fees_management_enabled && var.cidr_subnet_advanced_fees_management != null ? 1 : 0
+  source                                         = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v1.0.51"
+  name                                           = format("%s-afm-snet", local.project)
+  address_prefixes                               = var.cidr_subnet_advanced_fees_management
+  resource_group_name                            = azurerm_resource_group.rg_vnet.name
+  virtual_network_name                           = module.vnet.name
+  enforce_private_link_endpoint_network_policies = true
+
+#  service_endpoints                              = ["Microsoft.Web"]
+
+  delegation = {
+    name = "default"
+    service_delegation = {
+      name    = "Microsoft.Web/serverFarms"
+      actions = ["Microsoft.Network/virtualNetworks/subnets/action"]
+    }
+  }
+}
+
+# afm app configuration
+
+# app service: this is used only for PoC
+module "advanced_fees_management_app_service" {
```

Comment:
ok , but DR mention AKS

Reply:
yes, but there is a note that specifies we will use app service for time constraints (we have less than 10 days).
AKS will be the solution for the next future.